python/cugraph/components/connectivity_wrapper.pyx

```diff
@@ -34,20 +36,25 @@ def weakly_connected_components(input_graph):
     cdef uintptr_t graph = graph_wrapper.allocate_cpp_graph()
     cdef Graph * g = <Graph*> graph
 
-    if input_graph.adjlist:
+    if type(input_graph) is type_Graph and input_graph.adjlist:
         [offsets, indices] = graph_wrapper.datatype_cast([input_graph.adjlist.offsets, input_graph.adjlist.indices], [np.int32])
         [weights] = graph_wrapper.datatype_cast([input_graph.adjlist.weights], [np.float32, np.float64])
         graph_wrapper.add_adj_list(graph, offsets, indices, weights)
     else:
         [src, dst] = graph_wrapper.datatype_cast([input_graph.edgelist.edgelist_df['src'], input_graph.edgelist.edgelist_df['dst']], [np.int32])
         if input_graph.edgelist.weights:
             [weights] = graph_wrapper.datatype_cast([input_graph.edgelist.edgelist_df['weights']], [np.float32, np.float64])
+            if type(input_graph) is not type_Graph:
```

Comment:
Does this auto-symmetrize for WCC? All other raise exception in this case.

Should it be:
`raise Exception("Input graph must be undirected in WCC. Please use SCC or provide an undirected graph to WCC ")`